### ISSUE TITLE
[Bug] Conflict with PowerPoint, can't add notes for a slide when Dock Exposé is running

### ISSUE THREAD

**Issue by ftorelli165** [steventheworker/DockAltTab#23]
<details>
  <summary>Permissions reset</summary>

Right click is now blocked by the preview window. It looks like it's attempting to open, stutters, but can't get it to open.

Steps to reproduce the bug
1. Right click on open app

DockAltTab version: 2.8
macOS version: 13.2.1

I was able to reset all my permissions with your instructions, and it does appear to have solved this issue, but there's</details> another issue I noticed this morning in PowerPoint. I'll attach a screen recording where I'm trying to type notes into a PPT and you can see the mouse or focus is jumping around and I'm unable to actually select where I want to type in the notes section. When I turn dockalttab off it starts working again normally.
[Screen Recording 2023-05-24 at 9.15.04 AM.zip](https://github.com/steventheworker/DockAltTab/files/11555036/Screen.Recording.2023-05-24.at.9.15.04.AM.zip)

**Comment by steventheworker** [comment]
I think your permissions are goofing. [Here's how to reset them](https://github.com/steventheworker/DockAltTab/issues/19#issuecomment-1509900983).

It would also be interesting to know if simply updating the OS can fix this, since personally this usually only ever happens to me after updating the OS, which makes me wonder if it can undo this ghost permission issue.

**Comment by steventheworker** [comment]
I'm sure now that I've worked on a rebuild of this project that it's a PowerPoint bug. I'm adding Ubuntu's preview behavior (you can find it under pre-releases (v2.91.0))... it circumvents this issue, since it only reads mouse locations on click.

I am working on a workaround for the regular previews too, just posting with a proposed solution:

...since I'm already listening to clicks, I can just check if the active app is powerpoint and store the mouselocation of clicks that happen only on the the tricky element/input. when the mouse leaves the input, I can simulate the last mousedown/mousemove/mouseup that happened in the textarea (after reading the location), and do so until the user clicks outside the tricky element.

I don't know if this will work. Will update.

**Comment by steventheworker** [comment]
v2.91.0 has Ubuntu mode now, and I've since thought of another way of solving this, so later on it's possible I can get a workaround working for hover previews too!
